pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -92,95 +92,61 @@ For cloud installations, scaling out can be limited by resource quotas.
 
-  ```bash
-  kubectl kudo init --wait
-  ```
-
-- Download [kubeflow-1.4.0_1.3.0.tgz][download] tarball.
-<p class="message--note"><strong>NOTE: </strong>Starting with Kaptain 1.2.0, automatic profile creation on initial login is now disabled by default. See <a href="../../user-management">User Management</a> for more details.</p>
-
-- Set required configuration based on the target platform:
-  - When installing on Konvoy 1.x, add the following configuration to `parameters.yaml` file:
-  ```bash
-  cat >> parameters.yaml << END
-  dkpPlatformVersion: 1
-  installMinioOperator: true
-  END
-  ```
-- When installing on DKP 2.x, add the following configuration to `parameters.yaml` file:
-
-  ```bash
-  # set the OIDC Provider CA bundle
-  OIDC_PROVIDER_CA_BUNDLE=$(kubectl get secret kommander-traefik-certificate -n kommander -o jsonpath="{.data.ca\.crt}")
-
-  cat >> parameters.yaml << END
-  oidcProviderBase64CaBundle: ${OIDC_PROVIDER_CA_BUNDLE}
-  END
-  ```
-
-- Install Kaptain:
-  ```bash
-  kubectl kudo install --instance kaptain --namespace kubeflow --create-namespace \
-    ./kubeflow-1.4.0_1.3.0.tgz \
-    -P parameters.yaml
-  ```
-- If you would like to inject additional annotations to Kaptain's default `kubeflow-ingressgateway` `Gateway`, you can pass in the service annotations as parameters:
-  ```bash
-  kubectl kudo install --instance kaptain --namespace kubeflow --create-namespace \
-    ./kubeflow-1.4.0_1.3.0.tgz \
-    -P parameters.yaml \
-    -p kubeflowIngressGatewayServiceAnnotations='{"foo": "abc","bar": "xyz"}'
-  ```
-- Monitor the installation by running:
-  ```bash
-  kubectl kudo plan status --instance kaptain -n kubeflow
-  ```
-
-## Log in to Kaptain
-
-Once all components have been deployed, you can log in to Kaptain:
-
-- Discover the cluster endpoint and copy it to the clipboard.
-  If you are running Kaptain _on-premises_:
+### Create a Git repository for Kaptain
 
-  ```bash
-  kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].ip}"
-  ```
+1.  Refer to [air-gapped install instructions][airgapped_install], if you are running in air-gapped environment.
 
-  Or if you are running Kaptain on _AWS_:
-
-  ```bash
-  kubectl get svc kubeflow-ingressgateway --namespace kubeflow -o jsonpath="{.status.loadBalancer.ingress[*].hostname}"
-  ```
+1.  Adapt the URL of your Git repository:
 
-- Get the login credentials from Konvoy to authenticate:
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: kaptain-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels: 
+        kommander.d2iq.io/gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref: 
+        tag: 2.0.0
```

Comment:
this will only work after we created the tag (which is ok). @alembiewski any preferences on tag name?

Reply:
I'm wondering if we should put the DKP release version here (instead of the Kaptain version), as we do for the other `*-catalog-applications` repos: https://github.com/mesosphere/insights-catalog-applications/tags or https://github.com/mesosphere/dkp-catalog-applications/tags

Reply:
dang commited your change to early. I do not think we should have kaptain in syn cwith DKP, in fact other parts of DKP are eliminating this version link currently.